lib/ansible/modules/cloud/atomic/atomic_image.py

```diff
@@ -87,9 +103,38 @@ def core(module):
     image = module.params['name']
     state = module.params['state']
     started = module.params['started']
+    backend = module.params['backend']
     is_upgraded = False
 
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
+    out = {}
+    err = {}
+    rc = 0
+
+    if backend:
```

Comment:
I am just thinking is there any way to merge with backend and without backend? There are few command differences, but is it possible to modify so that we can merge?